modules/sdk-coin-opeth/src/lib/transactionBuilder.ts

```diff
@@ -13,6 +20,20 @@ export class TransactionBuilder extends EthTransactionBuilder {
     this.transaction = new Transaction(this._coinConfig, this._common);
   }
 
+  /**
+   * Returns the smart contract encoded data
+   *
+   * @param {string[]} addresses - the contract signers
+   * @returns {string} - the smart contract encoded data
+   */
+  protected getContractData(addresses: string[]): string {
+    const params = [addresses];
+    const resultEncodedParameters = EthereumAbi.rawEncode(walletSimpleConstructor, params)
+      .toString('hex')
+      .replace('0x', '');
+    return walletSimpleByteCode + resultEncodedParameters;
```

Comment:
May be we can add it to a property / method in the class to get this value which can help us abstract it further and avoid the need of redefining it each time

Reply:
I will address this in a follow up PR as part of this ticket WIN-1012